using/operating-a-star.md

```diff
@@ -27,7 +29,23 @@ Below are some handy operations for the use and maintenance of your star.
 
 ### DNS Proxying
```

Comment:
The "DNS Proxying" terminology used here in the past conflated the dynamic DNS records and TCP proxying that constituted arvo.network -- it's not really a thing. I'd change this title to "arvo.network DNS records", or something like that.